packages/front-end/components/PowerCalculation/PowerCalculationSettingsModal.tsx

```diff
@@ -213,60 +264,108 @@ const InputField = ({
         )}
       </>
     ),
-    min: minValue,
-    max: maxValue,
+    ...(c.type !== "boolean" ? { min: c.minValue, max: c.maxValue } : {}),
     className: clsx("w-50", isKeyInvalid && "border border-danger"),
     helpText: isKeyInvalid ? (
       <div className="text-danger">{helpText}</div>
     ) : undefined,
   };
 
   return (
-    <div className="col">
-      {isPercent ? (
+    <div className="col-4">
+      {c.type === "percent" && (
         <PercentField
           {...commonOptions}
           value={entryValue}
           onChange={(v) => form.setValue(`metrics.${metricId}.${entry}`, v)}
         />
-      ) : (
+      )}
+      {c.type === "number" && (
         <Field
           {...commonOptions}
           {...form.register(`metrics.${metricId}.${entry}`, {
             valueAsNumber: true,
           })}
         />
       )}
+      {c.type === "boolean" && (
+        <div className="form-group h-100">
+          <div className="row align-items-center h-100 mt-2">
+            <div className="col-auto">
+              <Toggle
+                id={`input-value-${metricId}-${entry}`}
+                value={entryValue}
+                setValue={(v) => {
+                  form.setValue(`metrics.${metricId}.${entry}`, v);
+                }}
+              />
+            </div>
+            <div>{title}</div>
+          </div>
+        </div>
+      )}
     </div>
   );
 };
 
 const MetricParamsInput = ({
   form,
   metricId,
+  engineType,
 }: {
   form: Form;
   metricId: string;
+  engineType: "bayesian" | "frequentist";
 }) => {
   const metrics = form.watch("metrics");
   // eslint-disable-next-line
   const { name, type: _type, ...params } = ensureAndReturn(metrics[metricId]);
+  const [showBayesian, setShowBayesian] = useState(false);
 
   return (
     <div className="card gsbox mb-3 p-3 mb-2 power-analysis-params">
       <div className="card-title uppercase-title mb-3">{name}</div>
```

Comment:
Can we make the name a link to the metric?

Reply:
Inside the modal? That seems like a weird user experience to go from a modal to a new page..